spec/errorTest/baseCase/error_spec.js

```diff
@@ -1,6 +1,6 @@
-describe('finding an element that does not exist', function() {
-  it('should throw an error', function() {
-    browser.get('index.html');
+describe('finding an element that does not exist', () => {
```

Comment:
As far as I can tell this file is never used - can we just remove it?

Reply:
Deleted.